### ISSUE TITLE
Missing data from manifest

### ISSUE THREAD

**Issue by geiseri** [jtbgroup/kodi-media-sensors#2]
I see this error on startup with new home assistant
```
2021-05-06 08:03:48 ERROR (MainThread) [homeassistant.loader] No 'version' key in the manifest file for custom integration 'kodi_media_sensors'. As of Home Assistant 2021.6, this integration will no longer be loaded. Please report this to the maintainer of 'kodi_media_sensors'
```

**Comment by jtbgroup** [comment]
Hi,

I just noticed it today too... I added the version in the manifest and published a relase 2.0.1.

**Comment by geiseri** [comment]
Perfect!
